src/cobald/daemon/config/python.py

```diff
@@ -1,5 +1,26 @@
-import include
+import pathlib
+import importlib.util
+import sys
+
+
+_loaded = -1
 
 
 def load_configuration(path):
-    return include.path(path)
+    """
+    Load a configuration from a module stored at ``path``
+
+    The ``path`` must end in a valid file extension for the appropriate module type,
+    such as ``.py`` or ``.pyc`` for a plaintext or bytecode python module.
+    """
+    global _loaded
+    current_index = _loaded = _loaded + 1
```

Comment:
What is the actual purpose of `_loaded`? Is it to create unique `module_name`s or do you also want to have a kind of counter on how many configurations have been loaded? I am just curious as in case of an error the counter is not being decremented.

Reply:
Yes, it's just to have a unique number. You are right that this isn't obvious, I'll think of another name.

Reply:
Changed the naming to indicate this is for identifying modules. Also using an ``itertools.count`` to get rid of the ``global`` (and possible race condition).